api/src/test/resources/org/openmrs/api/include/AdministrationServiceTest-globalproperties.xml

```diff
@@ -21,8 +21,8 @@
   <global_property property="valid.integer" property_value="1234" uuid="b7225607-d0c4-4e44-8be5-31e1ac7e1fda"/>
   <global_property property="valid.double" property_value="1234.54" uuid="b7225607-d0c4-4e44-8be6-31e1ac7e1fda"/>
 
-	<privilege privilege="Some Privilege For View Global Properties" description="This is a test privilege for view global properties" uuid="d979d066-15e6-467c-9d4b-cb575ef12345"/>
-	<privilege privilege="Some Privilege For Edit Global Properties" description="This is a test privilege for edit global properties" uuid="d979d066-15e6-467c-9d4b-cb575ef54321"/>
```

Comment:
I do not think we should change this file. Can't we do without these changes? The tests that expect these values should still do so.

Reply:
Not changing this could lead to problems with tests. For an example let's consider the following test.

https://github.com/openmrs/openmrs-core/blob/7d70503404031469b8e3fa98cc4dd39edb38e0c8/api/src/test/java/org/openmrs/api/AdministrationServiceTest.java#L636-L649

If we keep the privilege name as "Some Privilege For View Global Properties," this test will fail with an error message stating:

`org.openmrs.api.APIAuthenticationException: Privileges required: Get Global Properties`

We could mitigate the error by adding `Context.addProxyPrivilege(PrivilegeConstants.GET_GLOBAL_PROPERTIES);`
However, this goes against the test's purpose: to validate that only users with the appropriate privilege can access global properties.